src/main/shadow/cljs/ui/components/runtimes.cljs

```diff
@@ -8,6 +8,29 @@
 (defn age-display [since]
   (rel/format since))
 
+(defn build-lang
```

Comment:
For clarity maybe call this `build-lang-icon`?

Reply:
sure